### PR TITLE
Fix a warning about missing prototype on arm

### PR DESCRIPTION
**Pull request by bernd-edlinger**

### CODE REVIEW

File: crypto/include/internal/cryptlib.h [hunk 1]
```diff
@@ -74,6 +74,8 @@ FILE *openssl_fopen(const char *filename, const char *mode);
 void *openssl_fopen(const char *filename, const char *mode);
 # endif
 
+unsigned long OPENSSL_rdtsc(void);
```
Comment (mspncp) [inline crypto/include/internal/cryptlib.h]: Wasn't it decided in #4757 after a lengthy discussion that this should consistently be a uint32_t?  (see db26a403b4de69d7846656362e70f5083428)

Comment (mspncp) [reply crypto/include/internal/cryptlib.h]: correction: d807db26a403b4de69d7846656362e70f5083428

Comment (bernd-edlinger) [reply crypto/include/internal/cryptlib.h]: Maybe as another PR. This PR is just trying the fix the --strict-warnings build.
There are likely more issues, for instance sparcv9cap.c
defines "unsigned long OPENSSL_rdtsc(void)" without including "internal/cryptlib.h" first.

**Comment by bernd-edlinger** [comment]
Merged to 1.1.0
Thanks!

**Pull request closed without merging**